packages/core/src/servient.ts

```diff
@@ -164,14 +164,10 @@ export default class Servient {
         if (this.things.has(thingId)) {
             let serverPromises: Promise<boolean>[] = [];
             this.servers.forEach((server) => { serverPromises.push(server.destroy(thingId)); });
-            Promise.all(serverPromises).then(() => {
-                this.things.delete(thingId);
-                console.warn("[core/servient]", `Servient removed thing with ID '${thingId}'`);
-                return true;
-            }).catch((err) => {
-                console.warn("[core/servient]", `Servient removing thing with ID '${thingId}' failed`);
-                return false;
-            });
+            var results = (async () => await Promise.all(serverPromises));
```

Comment:
I think you need to call the function otherwise you are just creating a lambda function:
`(async () => await Promise.all(serverPromises))()`

Reply:
Are you sure?  I tried it locally and after destroy the thing is not available any longer with the current code

Reply:
I renamed it to `destroyThing()` and changed it to return a promise.
I think this resolved all issues... do you agree @relu91 ?

Reply:
yes! thank you 👍🏻